app/controllers/coronavirus_form/data_export_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::DataExportController < ApplicationController
+  def show
+    render plain: produce_csv(usage_statistics)
+  end
+
+private
+
+  def usage_statistics
+    questions = I18n.t("coronavirus_form.groups").map { |_, v| v[:questions] }.reduce(:merge)
+
+    start_date = Date.parse(sanitize(params[:start_date])).beginning_of_day
+    end_date = Date.parse(sanitize(params[:end_date])).end_of_day
+
+    results = {}
+    questions.each_key do |question|
+      question_text = questions.dig(question, :title)
+      counts = FormResponse.
+        where(created_at: start_date..end_date).
+        select(%{created_at::date, form_response -> '#{question}', COUNT(*)}).
+        group(%{created_at::date, form_response -> '#{question}'}).
+        pluck(%{created_at::date, form_response -> '#{question}', COUNT(*)}).
```

Comment:
These statements are throwing a fair few deprecation warnings:
```
DEPRECATION WARNING: Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s): "created_at::date, form_response -> 'able_to_leave', COUNT(*)". Non-attribute arguments will be disallowed in Rails 6.1. This method should not be called with user-provided values, such as request parameters or model attributes. Known-safe values can be passed by wrapping them in Arel.sql(). (called from block in usage_statistics at /Users/leenagupte/govuk/govuk-coronavirus-find-support/app/controllers/coronavirus_form/data_export_controller.rb:23)
```
There are a handful of examples in alphagov of using `Arel.sql` that we can "borrow" from: https://github.com/search?p=1&q=org%3Aalphagov+Arel.sql&type=Code

Reply:
Good find.  I'll use `Arel.sql` where required.